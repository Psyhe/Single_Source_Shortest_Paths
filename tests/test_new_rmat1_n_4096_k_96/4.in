4096 172 214
172 22 69
172 8 41
172 3104 174
172 297 181
172 1 107
172 1040 100
172 2122 187
172 3072 134
172 1096 104
172 1 26
172 531 114
172 19 171
172 834 1
172 157 44
172 1570 151
172 3152 91
172 18 8
172 665 148
172 151 151
172 2945 252
172 2570 178
172 68 250
172 8 36
172 1104 173
172 1086 159
172 2832 205
172 2176 105
172 8 187
172 1572 48
173 13 153
173 128 94
173 1035 160
173 384 172
173 555 115
173 2048 122
173 0 44
173 2 199
173 880 10
174 2702 40
174 2512 34
174 664 175
174 296 243
174 1025 2
174 72 41
174 272 93
175 64 115
176 20 23
176 2184 66
176 65 67
176 0 101
176 1088 153
176 2128 48
176 8 13
176 2106 25
176 2784 35
176 1344 243
176 532 48
176 1120 218
176 267 51
176 3616 179
176 320 72
176 2053 224
176 0 236
176 3594 249
176 1880 73
176 257 201
176 2 217
176 65 7
176 2693 101
176 1032 128
176 256 23
176 1040 85
176 0 249
176 4 101
176 70 189
176 162 79
176 1028 28
176 386 106
176 2625 136
176 160 224
176 1561 17
176 107 144
176 0 168
176 363 250
176 288 23
176 17 101
176 0 229
176 1036 164
176 1066 221
176 1060 10
176 512 178
176 1368 220
176 512 235
176 1346 63
176 3840 209
176 546 25
176 140 93
176 1560 175
176 1044 42
176 2380 158
176 2372 191
176 777 135
176 513 179
176 1024 236
176 512 61
176 2592 140
176 360 106
176 1794 30
176 1123 244
176 6 108
176 128 21
176 2660 193
176 338 75
176 3092 37
176 2898 65
176 192 207
176 1536 241
176 264 167
176 328 70
176 514 236
176 128 70
176 1026 238
177 3340 253
177 1280 9
177 320 26
177 360 234
177 514 238
177 64 246
177 146 190
177 1588 45
177 1541 187
177 320 31
177 128 138
177 2722 125
177 1152 73
177 260 182
177 3 182
177 1061 228
177 1 173
177 1097 17
177 2120 233
177 32 15
177 417 124
178 2048 126
178 0 102
178 77 60
178 3680 211
178 2076 142
178 2560 201
178 8 82
178 420 26
178 1169 248
178 0 59
178 0 193
178 4 72
178 2504 11
178 65 247
178 3080 203
178 768 222
178 2608 119
178 1155 185
178 0 72
178 0 239
179 3584 141
179 33 244
179 260 201
179 2050 141
179 1153 243
179 272 74
179 3098 191
179 2816 56
179 184 25
179 1032 182
180 1120 115
180 1040 12
180 2048 17
180 2306 116
180 1063 65
180 68 3
180 35 153
180 1889 224
180 10 162
180 849 37
180 1172 170
181 69 120
181 608 102
181 0 142
181 0 73
181 4 13
181 2883 9
181 578 227
181 344 182
182 544 171
182 515 3
182 1985 125
182 136 98
183 2069 203
184 1 7
184 352 244
184 76 249
184 2080 55
184 32 57
184 2048 147
184 216 81
184 81 14
184 832 132
184 2826 195
184 0 194
184 194 64
184 2816 111
184 283 214
184 179 25
184 2659 225
184 578 181
184 521 115
184 320 131
184 8 148
184 708 128
184 2309 99
184 1040 169
185 2128 122
185 3856 197
185 2 242
185 264 165
185 664 160
185 528 191
185 36 67
185 2565 221
186 1027 223
186 1857 225
186 1152 5
186 3078 240
186 2913 143
186 256 212
186 0 61
186 4 226
186 52 17
187 810 190
187 320 229
188 416 17
188 50 30
188 131 207
188 290 205
188 5 102
189 560 83
189 2048 59
189 256 145
189 2128 131
189 1025 219
190 32 111
190 1026 217
190 259 59
192 37 170
192 18 38
192 524 48
192 784 136
192 320 59
192 1092 241
192 385 56
192 800 1
192 608 238
192 176 207
192 512 156
192 386 178
192 3408 199
192 2188 18
192 2198 145
192 2576 157
192 2056 231
192 599 241
192 2 172
192 2165 154
192 1192 13
192 2050 44
192 10 43
192 450 179
192 3074 207
192 2240 249
192 512 14
192 48 124
192 3891 26
192 1088 68
192 347 167
192 27 56
192 16 82
192 1925 112
192 3680 54
192 290 15
192 1760 80
192 0 142
192 81 166
192 0 65
192 32 168
192 16 200
192 258 217
192 13 12
192 777 60
192 2048 234
192 802 255
192 1809 64
192 2586 240
192 1 10
192 2049 252
192 1088 227
192 544 140
192 1224 249
192 577 64
192 216 58
192 5 216
192 1171 112
192 72 210
192 3337 69
192 2560 80
192 94 213
192 1832 93
192 513 173
192 708 238
192 288 136
192 2114 184
192 2051 183
192 74 183
192 1304 251
192 128 155
192 32 124
192 646 202
192 2816 60
192 2561 25
192 1038 112
192 1834 180
192 2378 78
192 2560 68
192 9 67
192 34 207
192 2242 60
192 596 66
192 838 83
192 1169 172
192 42 72
192 1028 26
192 9 128
192 384 163
192 132 22
192 1545 8
192 3144 50
192 0 66
192 2 169
192 129 236
192 512 84
192 2560 56
192 0 70
192 1157 251
192 0 17
192 1872 143
192 1 31
192 1874 112
192 3136 63
192 546 77
192 80 7
192 2624 25
192 551 181
192 672 163
192 8 161
192 1154 100
192 128 188
192 1153 172
192 3122 68
192 320 252
192 85 115
192 2108 113
192 768 68
192 416 151
192 2096 211
192 1538 17
192 152 1
192 1554 229
192 2064 224
192 563 232
192 2049 241
192 264 88
192 1 190
192 1104 1
192 2450 215
192 40 226
192 864 248
192 2484 228
192 2116 105
192 2824 249
192 16 185
192 68 54
192 130 127
192 2562 172
192 1075 184
192 267 24
192 1284 120
192 672 249
192 136 5
192 393 196
192 1346 63
192 448 202
192 70 161
192 132 157
192 658 67
192 3659 16
192 129 237
192 2 117
192 1112 78
192 13 95
192 396 24
192 278 100
192 1092 158
192 1416 95
192 1480 27
192 66 57
192 2052 202
192 2560 175
192 96 4
192 24 211
192 5 242
192 3088 1
192 1024 171
192 2048 224
192 2208 105
192 2 162
192 525 95
192 290 70
192 2688 47
192 17 179
192 2080 10
192 1 81
192 1558 113
192 10 253
192 1045 174
192 706 61
192 32 227
192 3749 210
192 2048 25
192 897 144
192 1028 68
192 36 59
192 132 11
192 3072 88
192 289 87
192 2176 31
192 577 233
192 2064 168
192 1 236
192 2062 122
192 131 227
192 2050 6
192 580 51
192 24 37
192 2144 129
192 2080 173
192 1128 157
192 26 61
192 275 67
192 788 63
192 512 90
192 97 143
192 256 208
192 145 190
192 2 126
192 516 160
192 1350 122
192 256 82
192 0 31
192 3152 15
192 2176 84
192 320 95
192 2096 184
192 401 7
192 2048 66
192 12 157
192 2065 99
192 273 176
192 2244 198
192 517 239
192 529 172
192 4 164
192 1290 83
192 1056 37
192 2296 95
192 2218 217
192 124 167
192 40 65
192 640 194
192 168 197
192 2320 67
192 52 132
192 256 127
192 3106 65
192 51 100
192 33 244
192 515 238
192 854 135
192 1025 180
192 1152 113
192 0 7
192 2 163
192 133 111
192 2092 70
192 3072 231
192 68 121
192 3336 251
192 277 84
193 340 226
193 1796 153
193 164 171
193 260 80
193 14 24
193 2154 203
193 288 142
193 112 213
193 1040 192
193 40 54
193 1058 136
193 4 188
193 1696 183
193 512 37
193 2163 54
193 168 97
193 778 26
193 3361 114
193 771 242
193 10 91
193 2052 61
193 0 21
193 515 120
193 552 193
193 3204 179
193 1256 62
193 796 0
193 432 134
193 1040 181
193 705 57
193 2048 189
193 2434 172
193 2234 222
193 2576 11
193 2048 173
193 70 111
193 518 4
193 78 31
193 552 64
193 1028 241
193 548 143
193 776 238
193 864 139
193 306 163
193 2624 136
193 1 96
193 3074 189
193 1152 13
193 132 109
193 518 66
193 144 132
193 1024 223
193 1794 143
193 2064 114
193 12 184
193 128 172
193 416 184
193 138 193
193 1795 149
193 133 211
193 300 115
193 32 247
193 70 255
194 132 202
194 34 168
194 2098 218
194 217 146
194 3074 47
194 97 23
194 514 234
194 265 65
194 4 26
194 2308 101
194 2128 27
194 33 22
194 44 214
194 816 180
194 1 147
194 1345 132
194 512 188
194 130 84
194 0 46
194 1026 206
194 1034 124
194 184 64
194 16 183
194 128 52
194 2088 76
194 2656 106
194 2560 246
194 271 139
194 457 14
194 1296 210
194 1 152
194 544 205
194 1108 86
194 1545 97
194 3369 17
194 288 148
194 2208 205
194 1550 79
194 338 32
194 160 118
194 32 67
194 28 207
194 1 214
194 128 74
194 2600 110
194 1152 21
194 1029 213
194 292 135
194 264 157
194 2368 37
194 2689 27
194 270 59
194 0 174
194 1 133
194 2192 197
194 548 152
194 0 219
194 15 43
194 1169 169
194 2055 154
194 17 199
194 1056 133
194 269 103
194 3382 49
194 152 221
194 2305 156
194 560 73
194 0 207
194 3016 37
194 8 194
194 2 243
194 576 253
194 288 8
194 257 215
194 584 236
194 3 5
194 1376 218
195 2068 89
195 2592 116
195 2069 191
195 10 115
195 45 57
195 0 151
195 3264 112
195 144 128
195 2048 6
195 1028 130
195 0 140
195 2052 1
195 2048 100
195 2336 122
195 2052 34
195 5 115
195 516 236
195 16 161
195 19 46
195 2600 78
195 2818 103
195 2049 132
196 2649 32
196 264 77
196 331 173
196 68 165
196 593 71
196 1072 5
196 1666 236
196 1538 161
196 52 128
196 132 32
196 42 249
196 1312 94
196 265 4
196 2600 250
196 1 17
196 2064 246
196 2608 93
196 89 59
196 0 121
196 64 224
196 358 10
196 75 80
196 256 172
196 1290 242
196 2 253
196 3202 62
196 32 237
196 1536 71
196 9 66
196 1208 150
196 8 131
196 96 124
196 680 21
196 25 136
196 1858 171
196 256 120
196 0 98
196 272 152
196 2368 195
196 48 153
196 328 11
196 536 19
196 2657 176
196 562 171
196 2050 8
196 2369 34
196 1025 198
196 1804 131
196 1138 83
196 32 165
196 10 85
196 152 168
196 770 232
196 112 232
196 3464 251
196 2072 154
196 740 29
196 3074 50
196 44 62
196 321 183
196 14 253
196 66 93
196 1616 249
196 1056 7
196 2048 20
196 512 170
196 1536 203
196 389 236
196 2209 194
196 2180 184
196 2084 153
196 0 67
196 0 14
196 2 170
196 1048 236
196 2793 79
196 64 121
196 1 69
196 97 29
196 10 196
196 3652 220
196 65 50
197 2576 167
197 96 195
197 520 56
197 1024 191
197 1088 180
197 40 196
197 1028 121
197 48 143
197 16 44
197 3215 127
197 1152 97
197 136 72
197 1056 117
197 1025 189
197 2048 104
197 40 129
197 34 44
197 136 142
197 544 185
197 3596 96
197 2054 72
198 1160 127
198 2378 105
198 56 231
198 545 135
198 1024 16
198 513 101
198 1043 83
198 1368 99
198 640 112
198 3138 243
198 2314 93
198 3432 78
198 33 151
198 537 237
198 32 86
198 2952 168
198 2048 120
198 2112 178
198 520 209
198 1800 97
198 1616 61
198 16 1
198 512 206
198 514 179
198 1113 252
198 8 141
198 266 207
199 17 134
199 166 54
199 1032 227
199 1921 142
199 258 249
199 9 125
199 0 212
199 2584 150
199 1602 4
199 133 75
199 2944 166
200 1282 220
200 2120 6
200 1041 28
200 322 83
200 2880 80
200 1036 156
200 657 189
200 128 38
200 64 110
200 420 16
200 68 233
200 16 210
200 1185 109
200 57 245
200 144 82
200 2177 154
200 14 129
200 2819 56
200 1152 47
200 2073 2
200 2182 108
200 516 78
200 320 249
200 8 28
200 1026 193
200 262 149
200 528 82
200 148 24
200 2051 215
200 396 82
200 1120 7
200 2 207
200 3248 46
200 320 205
200 1760 164
200 576 18
200 1803 219
200 640 194
200 2 175
200 2049 145
200 2593 206
200 656 199
200 528 148
200 2720 147
200 2177 127
200 3328 170
200 257 151
200 1552 59
200 4 197
200 1 249
200 2324 82
200 520 37
200 3089 41
200 3840 93
200 512 156
200 912 232
200 848 36
200 5 16
200 1664 89
200 2048 6
200 513 91
200 65 19
200 1041 31
200 1281 112
200 1091 45
200 10 210
200 613 233
200 2560 23
200 2178 136
200 288 116
200 1187 125
200 8 79
200 1042 45
200 896 90
200 130 189
200 1734 205
200 1280 198
200 8 124
200 25 157
200 516 231
200 103 231
200 321 93
200 2068 222
200 2055 202
200 18 15
200 137 171
200 2772 67
200 0 64
200 3008 163
200 2368 232
200 288 214
201 352 114
201 2562 156
201 16 153
201 512 130
201 32 162
201 124 28
201 968 251
201 18 74
201 3174 68
201 0 133
201 512 127
201 1188 233
201 2506 172
201 4 120
201 1 233
201 657 94
201 276 101
201 608 125
201 64 254
201 593 29
201 392 94
201 1092 164
201 288 153
201 513 113
202 256 209
202 1054 47
202 2130 160
202 580 177
202 2572 88
202 1538 16
202 1153 210
202 1024 68
202 2306 90
202 61 168
202 80 192
202 2082 174
202 1540 3
202 1 69
202 8 248
202 3 95
202 2090 219
202 34 168
202 16 77
202 2716 181
202 0 180
202 256 50
202 1536 67
202 48 238
202 8 58
202 385 150
202 3588 19
203 1585 120
203 1296 96
203 1312 42
203 3081 109
204 2176 78
204 64 178
204 2057 69
204 1170 161
204 1024 19
204 2091 58
204 297 166
204 136 171
204 521 232
204 2304 28
204 258 216
204 1120 126
204 512 154
204 1032 231
204 0 4
204 264 93
204 66 136
204 514 251
204 3616 223
204 17 144
204 2408 65
204 160 44
204 3145 220
205 16 161
205 1024 195
205 66 232
205 5 251
205 524 186
205 24 59
205 1344 150
205 1184 228
206 2192 37
206 64 31
206 1048 80
206 2048 167
206 65 225
207 2176 74
207 2299 14
207 9 91
207 33 63
208 2569 27
208 2304 59
208 276 72
208 1136 125
208 2128 191
208 528 29
208 512 254
208 1664 252
208 1184 73
208 8 164
208 1773 94
208 645 96
208 2082 245
208 1032 205
208 1537 222
208 4 52
208 128 94
208 138 62
208 80 55
208 2564 156
208 212 125
208 258 43
208 12 17
208 16 217
208 1042 109
208 840 166
208 1259 126
208 2432 249
208 1136 236
208 268 135
208 1064 224
208 581 15
208 1 251
208 1032 80
208 2050 184
208 256 162
208 513 113
208 261 193
208 1264 121
208 1048 199
208 4 220
208 570 17
208 7 148
208 1538 115
208 1029 2
208 1928 93
208 385 201
208 2084 153
208 1160 144
208 17 102
208 264 113
208 12 112
208 391 59
208 164 53
208 3393 44
208 16 56
208 4 180
208 96 204
208 544 20
208 0 232
208 1302 93
208 33 207
208 3 156
208 1216 162
208 0 49
208 58 30
208 1026 181
208 586 234
208 280 131
208 0 224
208 17 127
208 9 134
208 577 75
208 1056 9
208 322 148
208 3120 69
208 2306 255
208 35 132
209 3264 84
209 9 29
209 290 242
209 16 49
209 1315 232
209 32 188
209 517 167
209 320 198
209 514 235
209 225 112
209 3072 49
209 260 113
209 3200 214
209 0 165
209 3084 96
209 2595 229
209 2109 19
209 1620 186
209 2212 52
209 8 96
209 2976 96
209 522 4
209 262 188
209 0 227
209 2577 185
209 2741 145
209 2177 215
209 512 203
209 393 198
209 8 24
209 2320 46
209 3073 141
209 262 19
209 1668 193
209 85 93
210 328 115
210 3336 31
210 2582 63
210 1552 6
210 592 145
210 1320 55
210 640 113
210 2176 81
210 3205 118
210 452 245
210 1027 23
210 1536 201
210 288 160
210 2052 192
210 448 131
210 2076 46
210 2050 224
210 1099 51
210 4 248
210 1288 121
210 3 163
210 260 4
210 2056 1
210 1452 131
210 475 219
210 2704 3
210 48 126
210 65 25
210 516 17
210 775 16
210 66 96
211 521 34
211 884 82
211 768 62
211 1536 75
211 2562 121
211 2 89
211 1792 72
211 787 13
211 323 58
211 5 104
212 2209 32
212 291 129
212 256 5
212 772 89
212 6 129
212 2976 78
212 1152 243
212 72 79
212 256 225
212 1025 46
212 1338 13
212 41 124
212 2064 231
212 2057 162
212 1046 241
212 4 133
212 0 7
212 65 162
212 2185 238
212 66 225
212 546 252
212 2 192
212 208 125
212 8 11
212 0 134
212 3809 214
212 3074 219
212 1202 77
213 1536 168
213 128 223
213 328 153
213 268 86
213 524 13
213 2272 101
213 2578 111
213 1025 7
213 7 164
213 38 60
214 26 86
214 516 128
214 1154 110
214 9 2
214 2052 172
214 530 98
214 2056 53
