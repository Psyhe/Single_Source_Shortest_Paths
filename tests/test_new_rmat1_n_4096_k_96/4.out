11
44
5
116
14
15
49
33
6
14
5
213
8
68
10
235
24
60
67
inf
4
9
9
3
10
46
3
18
7
60
30
42
4
61
32
64
13
11
4
27
6
10
6
